resources/js/main.js

```diff
@@ -52,18 +52,32 @@ function dataObjectUpdated() {
 }
 
 function removeItem() {
```

Comment:
function getcurrentProcessElement(thisElement) {
  var item = this.parentNode.parentNode;
  var parent = item.parentNode;
  var id = parent.id;
  var value = item.innerText;
  return {
    item,
    parent,
    id,
    value
  }
}
function upDateDomAfterRemove(parent, item) {
  parent.removeChild(item);
}

function upDateLocalStorage() {
  localStorage.setItem('todoList', JSON.stringify(data));
}

function upDateGlobalDate(id, value) {
  if (id === 'todo') {
    data.todo.splice(data.todo.indexOf(value), 1);
  } else {
    data.completed.splice(data.completed.indexOf(value), 1);
  }
}

function removeItem() {
  let currentProcessElement = getcurrentProcessElement(this);
  upDateDomAfterRemove(currentProcessElement.parent, currentProcessElement.item);
  upDateGlobalDate(currentProcessElement.id, currentProcessElement.value);
  upDateLocalStorage();
}